05_syntax_lab/01.py

```diff
@@ -1,5 +1,19 @@
-"""
-Write a program that reads 10 numbers from
-the user and prints the largest one
-"""
-
+"""
+
+write program that gets from user 10 numbers and prints the biggest
+
+
+"""
+
+
+max =0
+
+for i in range (1,11):

```

Comment:
Since you don't care about the value of i inside the loop, there's a nicer syntax for this:

```
for _ in range(10):
```